### PR TITLE
Fix - Account Language Selection || Fix language helper variable interpretation

### PR DESCRIPTION
**Pull request by AyakorK**
Hello ! 

Here's a PR to correct the fact that if we had only one locale, dropdown was still active -> Now he's disabled if we only have one and work properly if we have some more.

Another thing was to add some tests and to correct the bug of %{organization_name} interpretation that was not done. Now fixed and work also properly.

### CODE REVIEW

File: spec/system/account_spec.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@
 require "spec_helper"
 
 describe "Account", type: :system do
+  # If you want to try with only one locale -> Comment line 10 and uncomment line 7
```
Comment (Quentinchampenois) [inline spec/system/account_spec.rb:6]: Please remove comments here

File: spec/system/account_spec.rb [hunk 2]
```diff
@@ -38,11 +40,26 @@
       end
     end
 
+    describe "update locales" do
+      it "shows the list of locales" do
+        find("#user_locale").click
+        if organization.available_locales.size == 1
```
Comment (Quentinchampenois) [inline spec/system/account_spec.rb:46]: Please add a second case. 
First test should test with multiple locales
Second test should test with one locale

File: spec/system/account_spec.rb [hunk 3]
```diff
@@ -53,6 +70,12 @@
         within ".title-bar" do
           expect(page).to have_content("Nikola Tesla")
         end
+        if organization.available_locales.size > 1
```
Comment (Quentinchampenois) [inline spec/system/account_spec.rb:73]: Same here, avoid conditions in tests since this is not the same case

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]

File: app/views/decidim/account/show.html.erb [hunk 4]
```diff
@@ -11,7 +11,7 @@
       <%= f.text_field :nickname %>
       <%= f.email_field :email %>
 
-      <% if Decidim.available_locales.size > 1 %>
+      <% if @account.organization.available_locales.size > 1 %>
```
Comment (Quentinchampenois) [inline app/views/decidim/account/show.html.erb:14]: Prefer using `current_organization`

File: spec/system/account_spec.rb [hunk 5]
```diff
@@ -3,7 +3,6 @@
 require "spec_helper"
 
 describe "Account", type: :system do
-  # If you want to try with only one locale -> Comment line 10 and uncomment line 7
   # let(:organization) { create(:organization, available_locales: %w(en)) }
```
Comment (Quentinchampenois) [inline spec/system/account_spec.rb:6]: We should also remove this comment

File: spec/system/account_spec.rb [hunk 6]
```diff
@@ -41,13 +40,21 @@
...
-      it "shows the list of locales" do
-        find("#user_locale").click
-        if organization.available_locales.size == 1
-          expect(page).to have_css("#user_locale", text: "English")
-          expect(find("#user_locale")).to be_disabled
-        else
-          expect(page).to have_css("option", count: organization.available_locales.size)
+      context "when the user has one locale" do
+        it "is not possible to change locales, #user_locale is disabled" do
+          if organization.available_locales.size === 1
```
Comment (Quentinchampenois) [inline spec/system/account_spec.rb:45]: Please remove the `if` condition

File: spec/system/account_spec.rb [hunk 7]
```diff
@@ -41,13 +40,21 @@
...
+            expect(page).to have_css("#user_locale", text: "English")
+            expect(find("#user_locale")).to be_disabled
+          end
+        end
+      end
+
+      context "when the user has more than one locale" do
+        it "shows the list of locales" do
+          if organization.available_locales.size > 1
```
Comment (Quentinchampenois) [inline spec/system/account_spec.rb:54]: Please remove the `if` condition

File: spec/system/account_spec.rb [hunk 8]
```diff
@@ -70,11 +73,37 @@
...
-        if organization.available_locales.size > 1
-          within "#user_locale" do
-            expect(page).to have_content("Français")
+      end
+    end
+
+    describe "updating locale" do
+      context "when the user has one locale" do
+        it "is not possible to change locales, #user_locale is disabled" do
+          if organization.available_locales.size == 1
```
Comment (Quentinchampenois) [inline spec/system/account_spec.rb:82]: Please remove the `if` condition

File: spec/system/account_spec.rb [hunk 9]
```diff
@@ -70,11 +73,37 @@
...
+            expect(page).to have_css("#user_locale", text: "English")
+            expect(find("#user_locale")).to be_disabled
+          end
+        end
+      end
+
+      context "when the user has more than one locale" do
+        it "switches the locale to french" do
+          if organization.available_locales.size > 1
```
Comment (Quentinchampenois) [inline spec/system/account_spec.rb:91]: Please remove the `if` condition

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]

File: app/views/decidim/account/show.html.erb (hunk 4 above)
Comment (Quentinchampenois) [reply app/views/decidim/account/show.html.erb:14]: Deprecated since it is used as this below in Decidim's core

**Review by Quentinchampenois — APPROVED** [review]

**Comment by OuardaOUBAHRI** [comment]
Good work ! thank you

**Pull request merged**